tech_reports/MetalProfiler/metal-profiler.md

```diff
@@ -0,0 +1,349 @@
+# Metal Profiler
+
+## Quick Links
+- Tracy Profiler Repo: https://github.com/wolfpld/tracy
+- Tracy Documentation: https://github.com/wolfpld/tracy/releases/latest/download/tracy.pdf
+- Metal Fork of Tracy: https://github.com/tenstorrent-metal/tracy/tree/71d4c8d378b52af7da7012b9b595a61e9304f0bb
+
+## Introduction
+Tracy is an open-source C++ profiling tool with sampling and code instrumentation profiling capabilities. The profiled application is a client, and the profiler itself is a server (by default runs on port 8086). It was named this way because the client is a thin layer that just collects events and sends them for processing and long-term storage on the server. The fact that the server needs to connect to the client to begin the profiling session may be a bit confusing at first.
+
+## Things built from Tracy that are needed in tt-metal
```

Comment:
Do we need this here? Maybe we boil it down to a comment saying we are on 10 planning to move to 11

Reply:
I included that so folks know what tools are built and how metal's tracing is working internally. It was instructive for us to understand tracy-capture, tracy-profiler etc so we could modify it within our scripts

Reply:
what would be better is if I stick with metal's fork instead of 11

Reply:
I was just referring to this paragraph about 10 vs 11. The rest is very useful

Reply:
gotcha, I changed all the examples to use your forked repo instead.